modules/tinymce/src/themes/silver/test/ts/headless/modes/ScrollingContextTest.ts

```diff
@@ -34,6 +44,26 @@ describe('headless.modes.ScrollingContextTest', () => {
       const div = SugarElement.fromHtml('<div style="overflow: scroll;">A</div>');
       assert.isTrue(ScrollingContext.isScroller(div), 'Should be a scroller');
     });
+
+    it('TINY-9385: overflow-x: scroll - a scroller', () => {
```

Comment:
`overflow-x` and `overflow-y` have `hidden` and `clip` values https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-x. I suggest that more tests are needed to cover those values, and the cases where either `overflow-x` or `overflow-y` is `hidden` and `clip`.